pattern-hues.lic

```diff
@@ -54,16 +53,22 @@ class PatternHues
       /Something in the area is interfering/,
       /You try, but the dolphin choose/,
       /You should stop practicing/,
-      /You find it hard to concentrate on anything/
+      /You find it hard to concentrate on anything/,
+      /you suddenly realize that there is no custom color/,
+      /You'll interrupt your research!/
     ]
-    scripts_to_unpause = DRC.smart_pause_all
-    bput("prep cantrip pattern hues", "You are now prepared to cast the Pattern Hues cantrip.","You already have", "You should stop practicing")
+    scripts_to_unpause = DRC.safe_pause_list
+    unless scripts_to_unpause
+      DRC.message("Cannot pause, trying again shortly.")
+      return
+    end
+    DRC.bput("prep cantrip pattern hues", "You are now prepared to cast the Pattern Hues cantrip.","You already have", "You should stop practicing")
```

Comment:
I believe this is the bput that we want those missing matches added, and we'll want to stop before gesturing if we get failure, eg climbing, or researching, etc. perhaps a small case statement? when *failure => unpause and go back to the loop, else, gesture x or y.